src/main/webapp/WEB-INF/testNewRating.txt

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This works, just curious why the Mockito request does not work?

Reply:
Resolved this during our standup! :)